crates/vm/levm/src/opcode_handlers/environment.rs

```diff
@@ -171,6 +167,16 @@ impl VM {
             return Ok(OpcodeSuccess::Continue);
         }
 
+        if calldata_offset > current_call_frame.calldata.len().into() {
+            let data = vec![0u8; size];
```

Comment:
Done in [Move data definition to avoid doing it twice](https://github.com/lambdaclass/ethrex/pull/1421/commits/949c6ac0e7870604ee1d4868173886599f2d64b2)!